app/specs/controllers/StudentsIndexCtrl.spec.js

```diff
@@ -0,0 +1,50 @@
+// disable max-nested-callbacks linter for this test
+// it is good to have a good separation of contexts inside tests
+
+/* eslint-disable max-nested-callbacks */
+
+'use strict';
+
+define(['paw',
+'angular-mocks',
+'spec-utils',
+'api-responses',
+'controllers/StudentsIndexCtrl'],
+function() {
+  describe('Students Index Ctrl', function() {
+    beforeEach(module('paw'));
+
+    var $controller, $rootScope, controller, specUtilsService,
+    apiResponsesService;
+
+    beforeEach(inject(
+      function(_$controller_, _$rootScope_, specUtils, apiResponses) {
+        $controller = _$controller_;
+        $rootScope = _$rootScope_;
+        specUtilsService = specUtils;
+        apiResponsesService = apiResponses;
+        controller = $controller('StudentsIndexCtrl');
+      }));
+
+      describe('when reset button is clicked', function() {
+        beforeEach(function() {
+          controller.docket = 12345;
+          controller.firstName = 'John';
+          controller.lastName = 'Doe';
+          controller.resetSearch();
```

Comment:
Check that if removing this line the test should fail; if not, above lines are not applying the changes and you are not testing nothing. Just to be sure.